andela/AndelaEatsUI
# #161079274 Implement view engagement functionality

#### What does this PR do?
Add functionality for viewing engagements on the admin dashboard

#### Description of Task to be completed?
Fetch and replace static content with dynamic content

#### How should this be manually tested?
- Log in to the app
- Switch to the admin dashboard from the user profile dropdown menu
- Click on vendors on the side navigation menu
- You should see the engagements icon displayed

#### What are the relevant pivotal tracker stories?
- [161079274](https://www.pivotaltracker.com/n/projects/2178749/stories/161079274)

#### Background Context
Nil

#### Screenshots (If Applicable)
![screen shot 2018-11-14 at 9 50 49 am](https://user-images.githubusercontent.com/21138774/48486275-59041180-e81b-11e8-9093-0c87e2b2393d.png)
